src/Css.elm

```diff
@@ -1693,9 +1699,14 @@ pct value =
     Value (toString value ++ "%")
 
 
-{-| A unitless number. Useful with properties like [`flexGrow`](#flexGrow) which accept unitless numbers.
+{-| A unitless number. Useful with properties like
+[`flexGrow`](#flexGrow),
+[`z-index`](https://css-tricks.com/almanac/properties/z/z-index/),
```

Comment:
Nice, thanks for adding `zIndex` to this!

No need to change (I'll take care of it post-merge) but quick note: for "Useful with properties like" examples, I'd like to link to the `elm-css` docs (e.g. `[flexGrow](#flexGrow)`) instead of MDN or CSS Tricks.